### PR TITLE
fixed go vet report unbuffered channels to signal.Notify

### PR DESCRIPTION
**Pull request by zhanghe9702**
#### What type of PR is this?

/kind cleanup
#### What this PR does / why we need it:
make verify return failed when using new version vet tool
#### Which issue(s) this PR fixes:

Fixes #
make verify
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: hack/boilerplate/boilerplate.py [hunk 1]
```diff
@@ -218,7 +218,7 @@ def get_regexs():
     regexs["date"] = re.compile(get_dates())
     # strip // +build \n\n build constraints
     regexs["go_build_constraints"] = re.compile(
-        r"^(// \+build.*\n)+\n", re.MULTILINE)
+        r"^(// \+build.*\n|//go:build.*\n)+\n", re.MULTILINE)
```
Comment (wzshiming) [inline hack/boilerplate/boilerplate.py:221]: It looks like this line was fixed in your another PR https://github.com/kubernetes/kubernetes/pull/100361, revert ?

Comment (zhanghe9702) [reply hack/boilerplate/boilerplate.py:221]: no, this just fixed make verify

Comment (zhanghe9702) [reply hack/boilerplate/boilerplate.py:221]: sry, exist an redundancy commit

**Pull request closed without merging**